CefSharp.WinForms/Internals/MovingListener.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Copyright © 2014 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+using CefSharp.Internals;
+using System;
+using System.Drawing;
+using System.Runtime.InteropServices;
+using System.Windows.Forms;
+
+namespace CefSharp.WinForms.Internals
+{
+    internal class MovingListener : NativeWindow
```

Comment:
Overall I'd probably prefer to have a simple `NativeWindow` that forwards messages to `ChromiumWebBrowser`, so it can be used for other scenarios.